src/components/SingleReviewCard.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from "react"
+import { useParams } from "react-router-dom"
+import { getReviewsByReviewId } from "../utils/api"
+
+const SingleReviewCard = ({review, setReview}) => {
+    const {review_id} = useParams()
+    
+    useEffect(() => {
+        getReviewsByReviewId(review_id).then((reviewById) => {
+            setReview(reviewById)
```

Comment:
Could add a loading pattern so the user's not looking at a blank screen when waiting to load
If you're not sure how to add this take a gander at the bottom of the useEffect notes https://notes.northcoders.com/courses/js-front-end/useEffect 🙂